lib/pure/ioselects/ioselectors_epoll.nim

```diff
@@ -110,7 +110,7 @@ proc close*[T](s: Selector[T]) =
     raiseIOSelectorsError(osLastError())
 
 proc newSelectEvent*(): SelectEvent =
-  let fdci = eventfd(0, 0)
+  let fdci = eventfd(0, O_CLOEXEC)
```

Comment:
evenfd's man says:
> In Linux up to version 2.6.26, the flags argument is unused, and must be specified as zero.
https://linux.die.net/man/2/eventfd

Also, we can set `O_NONBLOCK` too here, and remove setNonBlocking.

Reply:
That's out of scope for this PR, sorry.

I'd like to focus on one change at a time, that way we can better trace any failure back to the source. But I do plan to get rid of `setNonBlocking()`.